governance.md

```diff
@@ -24,7 +24,9 @@ Governance
 3.  <a name="steering-committee-roles"></a>
     The Steering Committee makes all significant strategic and
     financial decisions for the Foundation.  Its members are also
-    expected to take on other responsibilities as needed.
+    expected to take on other responsibilities as needed, including 
+    reviewing appeals to decisions made by the Policy Committee regarding 
+    the Code of Conduct.
```

Comment:
Probably best to make this it's own list entry (3.c?).  I also [think][1] it might make more sense to put a single member of the committee in charge of monitoring board@ and translating appeals requests into motions.  Of course, any interested board member can help out, but explicitly assigning responsibility to a single board member avoids [diffusion of responsibility][2].

[1]: https://github.com/swcarpentry/board/issues/146#issuecomment-265659128
[2]: https://en.wikipedia.org/wiki/Diffusion_of_responsibility